tmc-plugin/src/fi/helsinki/cs/tmc/actions/RunTestsLocallyAction.java

```diff
@@ -20,12 +26,18 @@
     private TestRunHandler testRunHandler;
     private Project project;
 
+    private static final String GREEN_EYE = "testProjectGreen24.png";
+    private static final String RED_EYE = "testProjectRed24.png";
```

Comment:
I would prefer more descriptive file and variable names, like `ICON_TEST_RETURNABLE` and `icon_test_returnable.png` etc.